src/NServiceBus.AcceptanceTests/Mutators/When_using_outgoing_tm_mutator.cs

```diff
@@ -39,7 +39,6 @@ public Task MutateOutgoing(MutateOutgoingTransportMessageContext context)
                 {
                     context.OutgoingHeaders["HeaderSetByMutator"] = "some value";
                     context.OutgoingHeaders[Headers.EnclosedMessageTypes] = typeof(MessageThatMutatorChangesTo).FullName;
-                    context.OutgoingBody = Encoding.UTF8.GetBytes("<MessageThatMutatorChangesTo/>");
```

Comment:
hardly remember this. I think I said making this test to work with different serializes would require quite some work.

I remember something about ASQ changing serializer configuration. Does this interfere with `.Repeat.For(Serializers.Xml) `?